Formula/duktape.rb

```diff
@@ -1,7 +1,7 @@
 class Duktape < Formula
   desc "Embeddable Javascript engine with compact footprint"
   homepage "https://duktape.org"
-  url "https://duktape.org/duktape-2.5.0.tar.xz"
```

Comment:
why do switch to use github source?

Reply:
We absolutely don't but it enables brew livecheck to check for upstream updates.

Reply:
i see, that makes sense.